src/UnityExtension/Assets/Editor/GitHub.Unity/ApplicationCache.cs

```diff
@@ -10,25 +10,46 @@ namespace GitHub.Unity
 {
     sealed class ApplicationCache : ScriptObjectSingleton<ApplicationCache>
     {
-        [NonSerialized] private bool? val;
         [SerializeField] private bool firstRun = true;
+        [SerializeField] public string firstRunAtString;
+        [NonSerialized] private bool? firstRunValue;
+        [NonSerialized] public DateTimeOffset? firstRunAtValue;
 
         public bool FirstRun
         {
             get
             {
-                if (!val.HasValue)
+                if (!firstRunValue.HasValue)
                 {
-                    val = firstRun;
+                    firstRunValue = firstRun;
                 }
 
                 if (firstRun)
                 {
                     firstRun = false;
+                    FirstRunAt = DateTimeOffset.Now;
                     Save(true);
                 }
 
-                return val.Value;
+                return firstRunValue.Value;
+            }
+        }
+
+        public DateTimeOffset FirstRunAt
+        {
+            get
+            {
+                if (!firstRunAtValue.HasValue)
+                {
+                    firstRunAtValue = DateTimeOffset.Parse(firstRunAtString);
```

Comment:
Should you be passing a `CultureInfo` to these parse methods, or is it OK that they're being parsed using the `CurrentCulture`?

Reply:
I don't have a good answer here, I'm going to use a lifeline (i.e. @shana)

Reply:
https://github.com/github-for-unity/Unity/issues/448